documentation/publish/distribute-for-desktop-apps.md

```diff
@@ -38,7 +38,7 @@ Of these options, directing the user to install from AMO by offering a link is r
 - It avoids any issues with the installation process; the user will not get an interstitial messages during the installation of the add-on, find the add-on installed but disabled, or find that the add-on was not installed.
 - If you update the add-on, the new version will be automatically installed, once you have uploaded it to AMO.
 
-By contrast, sideloading using the standard extension folders or [Windows registry](/documentation/enterprise/enterprise-distribution/#installation-using-windows-registry) will require your desktop app to install any update to the add-on. Also, based on the default Firefox settings, the installation process will present the user with warnings (an interstitial message) or silently install the add-on but disable it. The worst case is that the installation will fail silently if [Firefox is setup to disable automatic installation](/documentation/enterprise/enterprise-distribution/#firefox-settings). You can update the [Firefox configuration](/documentation/enterprise/enterprise-distribution/#firefox-settings) to avoid these issues, but that is not recommended.
+By contrast, sideloading using the [standard extension folders](/documentation/publish/distribute-sideloading/#standard-extensions-folider) or [Windows registry](/documentation/enterprise/enterprise-distribution/#installation-using-windows-registry) will require your desktop app to install any update to the add-on. Also, based on the default Firefox settings, the installation process will present the user with warnings (an interstitial message) or silently install the add-on but disable it. The worst case is that the installation will fail silently if [Firefox is setup to disable automatic installation](/documentation/enterprise/enterprise-distribution/#firefox-settings). You can update the [Firefox configuration](/documentation/enterprise/enterprise-distribution/#firefox-settings) to avoid these issues, but that is not recommended.
```

Comment:
looks like there's a typo here, but let's use the id that's in distribute-sideloading. 
- folider --> folders 
- extensions --> extension

Reply:
typos are the worst :/